wiki/Game_modifier/Double_Time/id.md

```diff
@@ -0,0 +1,61 @@
+---
+tags:
+  - double time
+  - double-time
+  - DT
+  - mod
+  - game modifier
+---
+
+# Double Time (mod)
+
+![DT mod icon](/wiki/shared/mods/DT.png "Ikon mod double time (DT)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+*Lihat juga: [Nightcore (mod)](/wiki/Game_modifier/Nightcore)*
+
+## Tentang
+
+- Singkatan: DT
+- Tipe: Peningkat Kesulitan
+- Pengali skor:
+  - ![][o!s]: 1.12x
+  - ![][o!t]: 1.12x
+  - ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- Tombol shortcut default: `D`
+- Caption: `Zoooooooooom.`
+- Mode-mode permainan yang sesuai: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- Variasi: [Nightcore](/wiki/Game_modifier/Nightcore)
+
+## Dekripsi
+
+*Catatan: Cara yang digunakan untuk mempercepat lagu tersebut tidaklah menaikkan nada nya.*
+
+Mod **Double Time** adalah sebuah [game modifier](/wiki/Game_modifier) yang meningkatkan kesulitan sebuah [beatmap](/wiki/Beatmap) dengan meningkatkan kecepatan keseluruhan nya (BPM) sebesar 150% (1.5x), mengurangi durasi lagu itu sebesar 33%, dan meingkatkan [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty), dan [health (HP)](/wiki/Beatmapping/Health).
+
+Double time adalah sebuah mod yang sering dianggap sebagai salah satu mod terbaik yang digunakan untuk memperoleh banyak [performance points](/wiki/id/Performance_points) pada tingkat kesulitan yang lebih mudah di [osu!standard](/wiki/Game_mode/osu!).
+
+### osu!taiko
+
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), hit leniency dikurangi secara besar saat Double Time dinyalakan karena osu!taiko sudah mempunyai overall difficulty yang ketat bersama dengan slider tick yang lebih sulit untuk dikenai. Karena ini, mod Double time sering dianggap sebagai mod yang paling sulit di osu!tako dan jarang digunakan.
+
+### osu!catch
+
+Pada [osu!catch](/wiki/Game_mode/osu!catch), tidak terdapat overall difficulty untuk ditingkatkan. Karena itu, mod ini hanya meningkatkan BPM dan AR. Hasilnya, mod ini hanya bernilai 1.06x pengali skor (daripada 1.12x pengali dibandingkan dengan mod-mod lainnya).
+
+Meskipun mod ini meningkatkan hyperdaah leniency dengan besar, mengakibatkan fruit membutuhkan hyperdash yang kadang-kadang hampir mustahil untuk diraih.
+
+## Trivia
+
+- Saat mod Double Time dinyalakan, nilai `Length`, `BPM`, dan `Objects` akan bewarna merah-muda dengan nilai barunya. (Digambarkan dibawah.)
+  - Nilai `Objects` akan tetap bewarna merah-muda walaupun tidak ada perubahan yang terjadi kepada nilainya.
+- Nilai `CS`, `AR`, dan `HP` akan mempunyai sebuah segitiga dalam bentuk superscript disamping nilai mereka untuk menandakan peningkatan kecil pada nilai mereka. (Digambarkan dibawah.)
```

Comment:
```suggestion
- Nilai `CS`, `AR`, dan `HP` akan mempunyai sebuah segitiga kecil dalam bentuk superscript dibawah nilai berwarna merah-muda yang menandakan peningkatan kecil pada nilai. (Digambarkan dibawah.)
```

Reply:
actually it's located below it. see pic

Reply:
it is located next to it...?

"AR=9▲, OD: 7.5▲, HP: 6▲"